src/routes/tcg/_tcg.svelte

```diff
@@ -256,7 +256,7 @@
   }
 
   function swapCharacterCardPos(from, to) {
-    const keys = Object.keys(deck.characters);
```

Comment:
Typescript won't allow this to compile, allegedly the next line overwrites this variable. No idea why, this wasn't previously an issue